src/vsphere_cpi/lib/cloud/vsphere/config.rb

```diff
@@ -29,6 +29,10 @@ def use_policy_api?
     def policy_api_migration_mode?
       !!policy_api_migration_mode
     end
+
+    def allow_overwrite?
+      allow_overwrite || true
```

Comment:
The `|| true` here will prevent anyone from overriding the default value. It will continue always setting the `X-Allow-Overwrite` header.

Reply:
nice, bad code allowed by a bad test. 🤦